php-mode.el

```diff
@@ -24,7 +24,7 @@
 
 ;; This file is distributed in the hope that it will be useful,
 ;; but WITHOUT ANY WARRANTY; without even the implied warranty of
-;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
```

Comment:
Do not remove two spaces after period.
Please use [purcell/flycheck-package: Flycheck checker for elisp package metadata](https://github.com/purcell/flycheck-package).